### DISCUSSION TITLE
Skywalking Pods Failing

### DISCUSSION THREAD

**Discussion by AjinThankachan** [apache/skywalking#12049] [Q&A]
**Description**
I have tried to set up the Skywalking in Kubectl and 2 pods are failing and it is not able to connect. 

**Method Followed**

Installing the 4.5.0 version with Postgresql using the following command

```
helm install "%SKYWALKING_RELEASE_NAME%" oci://registry-1.docker.io/apache/skywalking-helm --version "%SKYWALKING_RELEASE_VERSION%" -n "%SKYWALKING_RELEASE_NAMESPACE%" --set oap.image.tag=9.6.0 --set oap.storageType=postgresql --set ui.image.tag=9.6.0 --set elasticsearch.enabled=false --set postgresql.enabled=true
Pulled: registry-1.docker.io/apache/skywalking-helm:4.5.0
Digest: sha256:5485fb9d95a033d7ac0d764b652487c3d071f4b5f2b9441e963734a0843e12e0
```
**Pods status**
![image](https://github.com/apache/skywalking/assets/25667606/5a2de417-5337-4a5a-8c95-392f5aa83795)
**Values.yml file**

[values.txt](https://github.com/apache/skywalking/files/14735606/values.txt)

**Getting Error in the POD when doing the description**
**![image](https://github.com/apache/skywalking/assets/25667606/6545512e-85ba-46ca-acae-552d494e0f98)**
![image](https://github.com/apache/skywalking/assets/25667606/63d156b1-1bc6-45f7-8d28-a4bfb80556ed)
![image](https://github.com/apache/skywalking/assets/25667606/90b230dd-c868-4e3c-ae7c-291c5e675316)

In the error I can see that for the posgress I have given the localhost as the IP then also it didn't take anything. ANd the IP it is showing in the error in the last image is not the one I have configured/not my IP. 

Is there any way i can resolve this.I think the issue is with the configuration. I am just started in Kubectl so don't have advanced knowledge in this. Also can you suggest any other thing that i need to do than the once i have executed already. I appreciate the time and effort to answer this in advance.

**Comment by kezhenxu94** [comment]
This is a question full of very common-sense errors, I'd rather suggest you go through the basic knowledge of Kubernetes, containers, and some well-known ports that common services use. To list some

- `127.0.0.1` inside the Pod is the Pod per se, not your local machine running kubectl
- the error message in the last screenshot is not what you understand, it's checking skywalking OAP healthiness, not checking you Postgres'

This is way far from where you can make it work, in my opinion, don't expect the community to help you solve the very basic issues, which is out of our scope and we don't have the efforts.

**Reply by AjinThankachan** [reply]
Thanks for the replay. So you are saying like there is nothing to do at the configuration end and no need to bother about the jdbc URL in the given image? And since rest of the pods are up the process i am following is also right?

![image](https://github.com/apache/skywalking/assets/25667606/6a60d7d4-ac4b-4a21-9930-625225af77e6)

**Reply by kezhenxu94** [reply]
> Thanks for the replay. So you are saying like there is nothing to do at the configuration end and no need to bother about the jdbc URL in the given image? And since rest of the pods are up the process i am following is also right?
> 
> 
> 
> ![image](https://github.com/apache/skywalking/assets/25667606/6a60d7d4-ac4b-4a21-9930-625225af77e6)
> 
> 

If everything is defaults they should be up and running. Only customize the configurations when you play well with the defaults and get yourself familiar with the defaults.
